README.md

```diff
@@ -253,6 +253,30 @@ struct directly to Bamboo anywhere it expects an address. See the
 [`Bamboo.Email`] and [`Bamboo.Formatter`] docs for more information and
 examples.
 
+## Handling Headers
+
+Email headers may be added to an email by adding them directly to the struct,
```

Comment:
Instead of describing the different ways to add headers in words, how do you feel about including examples instead? I find those to be a little bit easier to use when getting started with a project.

Also do you think we should recommend modifying the `%Bamboo.Email{}` struct in the docs? I feel like we should direct people to the utility functions we provide in case we someday change how the struct looks internally.

<hr />

## Handling Headers

There are two main ways to add headers to an email:

1. By passing a `headers` key to `Email.new_email/1`:

   ```elixir
    Email.new_email(%{headers: {"key" => "value"}})
   ```

1. By using `Email.put_header/4`:

    ```elixir
    %{}
    |> Email.new_email()
    |> Email.put_header("key", "value")
    ````

When adding headers via `Email.put_header/4` two strategies exist for adding headers of the same key. 
...

Reply:
👍